<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hello</class>
 <widget class="QMainWindow" name="hello">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>hello</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background:url(\Shreyam\download.jpg);
</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>731</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:green;
border-radius:40px;
background:lightgreen;
</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>250</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:red;
color:Black;
border-radius:20px;</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>250</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:red;
color:Black;
border-radius:20px;</string>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>USERNAME</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="username">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>60</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background:skyblue;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PASSWORD</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="password">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>110</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background:skyblue;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::PasswordEchoOnEdit</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nepali</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:40px;

background:pink;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHELLO">
    <property name="title">
     <string>HELLO</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuwow">
    <property name="title">
     <string>wow</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuok">
    <property name="title">
     <string>ok</string>
    </property>
   </widget>
   <addaction name="menuHELLO"/>
   <addaction name="menuwow"/>
   <addaction name="menuok"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>username</tabstop>
  <tabstop>password</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>hello</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>password</sender>
   <signal>returnPressed()</signal>
   <receiver>hello</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
